drafts/developer-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -0,0 +1,58 @@
+include::ROOT:partial$variables.adoc[]
+
+= Data Privacy Console
+
+The Data Privacy Console enables you to access the overview of your personal data, request data deletion or data export.
+
+This feature allows all users to browse locations and contents of their private data and make requests concerning the treatment of this data (e.g. Delete, Update), which are then escalated to the <<data-privacy-officer, *data privacy officer (DPO)*>> Console.
```

Comment:
Try to avoid `This feature` and refer to the feature by its name especially when it occurs in a different paragraph.